kerchunk/tests/test_hdf.py

```diff
@@ -331,3 +334,38 @@ def test_inline_threshold():
         fn, inline_threshold=1e9
     ).translate()
     assert inline_0 != inline_1_million
+
+
+def has_visititems_links():
+    try:
+        h5py.Group.visititems_links
+    except AttributeError:
+        return False
+    return True
```

Comment:
```suggestion
def has_visititems_links():
    return hasattr(h5py.Group, "visititems_links")
```

May as well use this in the main code too

Reply:
Good idea, thank you for the suggestions